docs/source/develop/api/queries.rst

```diff
@@ -58,11 +58,23 @@ After ordering is specified, pagination can be executed:
 .. code-block:: proto
 
     message TxPaginationMeta {
-        uint32 page_size = 1;
-        oneof opt_first_tx_hash {
-            string first_tx_hash = 2;
-        }
-        Ordering ordering = 3;
+      uint32 page_size = 1;
+      oneof opt_first_tx_hash {
+        string first_tx_hash = 2;
+      }
+      oneof opt_first_tx_time {
+        google.protobuf.Timestamp first_tx_time = 3;
+      }
+      oneof opt_last_tx_time {
+        google.protobuf.Timestamp last_tx_time = 4;
+      }
+      oneof opt_first_tx_height {
+        uint64 first_tx_height = 5;
+      }
+      oneof opt_last_tx_height {
+        uint64 last_tx_height = 6;
+      }
+      Ordering ordering = 7;
```

Comment:
These breaks current API. We should save new API compatible with older clients. New parameters should be optional and ordering (numbers of filds shouldd stay the same)

Reply:
You mean to place them after ordering parameter?

Reply:
I mean number after assign must be saved the same 
```suggestion
      Ordering ordering = 3;
```

Reply:
It is restored, so I'm marking its as resolved